### ISSUE TITLE
GLGAN：reproduction

### ISSUE THREAD

**Issue by tyran721** [sstary/SSRS#79]
<img width="574" alt="2732cd6370ac7d4a82bbeed321d52b4" src="https://github.com/user-attachments/assets/20b586d9-64fc-4816-b55c-c4a2f9d54973" />

Dear authors, your work is excellent, but I encountered some issues during the reproduction process. The results I obtained when reproducing the GLGAN work have not been satisfactory. The parameters, dataset, and pretrained models are all consistent with those in your paper. I would like to ask if there might be any potential issues that I have overlooked. For example, the mIoU for PIRRG2VIRRG in my results is approximately 0.5552, which is quite different from the value reported in the paper.

**Comment by sstary** [comment]
Could you please send the complete log to my email?
If all the hyperparameter, train-test set partitions agree with mine, then this MIoU is abnormal.
